Challenge-4/Bot_2/src/agent.spec.ts

```diff
@@ -2,81 +2,103 @@ import {
   MockEthersProvider,
   TestTransactionEvent,
 } from "forta-agent-tools/lib/test";
+
 import {
   ethers,
   Finding,
   FindingSeverity,
   FindingType,
   HandleTransaction,
 } from "forta-agent";
+
 import { provideInitialize, provideHandleTransaction } from "./agent";
 import { createAddress } from "forta-agent-tools";
 import Helper from "./helper";
-import { getAlerts } from "forta-agent";
-
-jest.mock("forta-agent", () => ({
-  ...jest.requireActual("forta-agent"),
-  getAlerts: jest.fn(),
-}));
+import { NetworkManager } from "forta-agent-tools";
+import { NetworkData } from "./types";
 
 describe("Compound test suite for lending and borrowing", () => {
   let mockProvider = new MockEthersProvider();
   let handleTransaction: HandleTransaction;
   let txEvent: TestTransactionEvent;
```

Comment:
You could make this of the type `TransactionEvent`.

Reply:
I mean that `txEvent` can be of type `TransactionEvent`. I.e., you can declare it as `let txEvent: TransactionEvent`. This is because this is the type that the bot actually expects and not `TestTransactionEvent`.

This is just a practice so that it adheres to what is actually expected. 🙂